cloudify_marketplace/resources/cloudify_security_reconfiguration

```diff
@@ -0,0 +1,267 @@
+#! /usr/bin/env python
+import json
+import logging
+import logging.handlers
+import os
+import re
+import shutil
+import subprocess
+import time
+import yaml
```

Comment:
Ah I see. I assumed the split was different (i.e. other stuff | cloudify stuff). I'll fix